community-operators/konveyor-forklift-operator/2.0.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,7 @@
+annotations:
+  operators.operatorframework.io.bundle.channels.v1: release-v2.0
+  operators.operatorframework.io.bundle.channel.default.v1: release-v2.0
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: konveyor-forklift-operator
```

Comment:
can you add 
`
com.redhat.openshift.versions: "=v4.7"
`

More info about versions: https://redhat-connect.gitbook.io/certified-operator-guide/ocp-deployment/operator-metadata/bundle-directory/managing-openshift-versions

Reply:
@mvalarh Done